src/test/java/uk/gov/pay/card/it/resources/CardIdResourceITest.java

```diff
@@ -68,6 +70,34 @@ public void shouldFindTestCardInformation() {
                 .body("corporate", is(false));
     }
 
+    @Test
+    public void shouldErrorWhenShortCardNumberProvided() {
+        getCardInformation(buildCardNumberFromPrefixAndLength("2221", CARD_RANGE_LENGTH - 2))
+                .statusCode(422)
+                .contentType(JSON)
+                .body("errors[0]", is(String.format("cardNumber size must be between %s and 19", CARD_RANGE_LENGTH)));
+    }
+
+    @Test
+    public void shouldErrorWhenOneCharTooShortCardNumberProvided() {
```

Comment:
this test is very similar to the one above, you can parameterise these to DRY this up a bit

Reply:
Updated, thanks.